pkg/proxy/iptables/proxier.go

```diff
@@ -1321,6 +1324,26 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 	proxier.portsMap = replacementPortsMap
+	// Delete conntrack for all UDP entries
+	proxier.clearUdpConntrackForPort(0)
```

Comment:
This is what I understood from your comment:
" ... You might also do it way at the end of this function when we close nodeports we don't want any more."

Now I removed it